### PR TITLE
routes updated with woas & bookings + path renamed

### PR DESCRIPTION
**Pull request by 0MY**
idem title

### CODE REVIEW

**Comment by Kevin33000** [comment]
ok c'est cool on merge !

**Comment by 0MY** [comment]
@Kevin33000 ok dude

**Comment by Muriel30** [comment]
Super ! Allez go !

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
   devise_for :users
-
+  resources :woas, only: [:index, :show], path: "oeuvres-art" do
+    resources :bookings, only: [:new, :create, :show], path: "reservations"
+  end
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```
Comment (0MY) [inline config/routes.rb]: Poor comment

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
   devise_for :users
-
```
Comment (Kevin33000) [inline config/routes.rb]: hgjkhg

Comment (0MY) [inline config/routes.rb:4]: la pô compris

Comment (0MY) [inline config/routes.rb:4]: this is a new conversation

**Review by 0MY — COMMENTED** [review]
review summary

File: config/routes.rb (hunk 2 above)
Comment (Muriel30) [reply config/routes.rb:4]: About the repartition of work... ?

**Comment by 0MY** [comment]
Review Done

**Comment by 0MY** [comment]
Go back and forth again

**Comment by Kevin33000** [comment]
cool on merge ?

**Pull request merged**